### PR TITLE
[action] add symbols and booleans in modify_services action

### PR DESCRIPTION
**Pull request by MartinGonzalez**
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary. Action description updated.

### Motivation and Context
This changes allow users to use Symbols & Booleans as parameters in `service` hash in `modify_services_action`

Related to: https://github.com/fastlane/fastlane/issues/14202

### Description
- Added Symbols & Booleans as valid parameters in `modify_services_action`

Previous `modify_services` example:

```
    modify_services(
      app_identifier: 'com.company.product',
      team_id: 'A255399',
      services: {
        push_notification: 'on',
        associated_domains: 'off'
      }
    )
```

Proposed `modify_services` example (keeping previous example valid):

```
    modify_services(
      app_identifier: 'com.company.product',
      team_id: 'A255399',
      services: {
        push_notification: :on,
        associated_domains: :off
      }
    )
```
or
```
    modify_services(
      app_identifier: 'com.company.product',
      team_id: 'A255399',
      services: {
        push_notification: true,
        associated_domains: false
      }
    )
```

How did I test it?
- Tested locally with a project following [How to test your local changes](https://github.com/fastlane/fastlane/blob/master/Testing.md#testing-your-local-changes)

### CODE REVIEW

**Comment by MartinGonzalez** [comment]
I signed it!

**Comment by MartinGonzalez** [comment]
I just signed the CLA

**Comment by MartinGonzalez** [comment]
I think I have problems with CLA. I just signed the CLA and the commit author email is the one signed there too.

**Comment by MartinGonzalez** [comment]
d

**Comment by MartinGonzalez** [comment]
CLA

File: fastlane/lib/fastlane/actions/modify_services.rb [hunk 1]
```diff
@@ -12,8 +12,8 @@ def self.run(params)
           require 'produce/service'
           services = params[:services]
 
-          enabled_services = services.reject { |k, v| v == 'off' }
-          disabled_services = services.select { |k, v| v == 'off' }
+          enabled_services = services.select { |_k, v| v == :on || v == 'on' || v == true }.map { |k, v| [k, 'on'] }.to_h
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/modify_services.rb]:
```suggestion
          enabled_services = services.select { |_k, v| v.to_sym == :on || v == true }.map { |k, v| [k, 'on'] }.to_h
```

File: fastlane/lib/fastlane/actions/modify_services.rb [hunk 2]
```diff
@@ -12,8 +12,8 @@ def self.run(params)
...
+          disabled_services = services.reject { |_k, v| v == :on || v == 'on' || v == true }.map { |k, v| [k, 'off'] }.to_h
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/modify_services.rb]:
```suggestion
          disabled_services = services.reject { |_k, v| v.to_sym == :on || v == true }.map { |k, v| [k, 'off'] }.to_h
```

File: fastlane/lib/fastlane/actions/modify_services.rb [hunk 3]
```diff
@@ -158,9 +158,12 @@ def self.example_code
...
             services: {
               push_notification: "on",
-              associated_domains: "off"
-            }
-          )'
+              associated_domains: "off",
+              wallet: :on,
+              apple_pay: :off,
+              data_protection: true,
+              multipath: false
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/modify_services.rb]: How come we are adding some of these in here? 🙃

**Review by joshdholtz — CHANGES_REQUESTED** [review]
A few small cleanups and a question here! But can we also get some tests added as well? 😊

File: fastlane/lib/fastlane/actions/modify_services.rb (hunk 3 above)
Comment (MartinGonzalez) [reply fastlane/lib/fastlane/actions/modify_services.rb]: I put an example of each value type, the way you can enable or disable services

**Comment by MartinGonzalez** [comment]
> A few small cleanups and a question here! But can we also get some tests added as well? 😊

I did not create this action, but I would love to do some test. I'm new in fastlane I do not know how to test an action. Should I simple create a double of the action class just like any ruby class?

**Comment by joshdholtz** [comment]
@MartinGonzalez I push a commit into your branch that added tests! Hope that's okay 😊

**Review by joshdholtz — APPROVED** [review]
Thanks for making these additions! It always confused me on why this couldn't be symbols or booleans 😊

**Comment by MartinGonzalez** [comment]
@joshdholtz Nice! You tested over an existing account or fastlane just fake it?

**Pull request merged**